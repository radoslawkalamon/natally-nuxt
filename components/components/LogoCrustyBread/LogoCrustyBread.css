@keyframes logo-crusty-bread-gradient {
  0% {
    background-position: 0% 0%;
  }

  50% {
    background-position: 100% 100%;
  }

  100% {
    background-position: 0% 0%;
  }
}

.logo-crusty-bread {
  animation: logo-crusty-bread-gradient 20s ease-in-out infinite;
  background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
  background-size: 400% 400%;
  display: inline-block;
  fill: var(--color-background-2);
  font-size: 0;
  line-height: 0;
}
