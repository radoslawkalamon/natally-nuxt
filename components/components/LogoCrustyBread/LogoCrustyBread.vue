<template v-once>
  <svg
    class="logo-crusty-bread"
    data-test="components-logo-crusty-bread"
    width="56"
    height="128"
    viewBox="0 0 56 128"
    fill="none"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M20 0H0V20C0 17.7909 1.79086 16 4 16C6.20914 16 8 17.7909 8 20V30.5279C8 32.043 8.85601 33.428 10.2111 34.1056C12.8708 35.4354 16 33.5014 16 30.5279V4C16 1.79086 17.7909 0 20 0Z"
    />
    <path
      d="M56 20C56 17.7909 54.2091 16 52 16C49.7909 16 48 17.7909 48 20V30.5279C48 32.043 47.144 33.428 45.7889 34.1056C43.1292 35.4354 40 33.5014 40 30.5279V12C40 9.79086 38.2091 8 36 8C33.7909 8 32 9.79086 32 12V38.5C32 40.7091 30.2091 42.5 28 42.5C25.7909 42.5 24 40.7091 24 38.5V4C24 1.79086 22.2091 0 20 0H56V20Z"
    />
    <path
      d="M56 87.0557C56 90.0859 54.288 92.856 51.5777 94.2111L36.4223 101.789C33.712 103.144 32 105.914 32 108.944V124C32 126.209 30.2091 128 28 128C25.7909 128 24 126.209 24 124V108.944C24 105.914 22.288 103.144 19.5777 101.789L4.42229 94.2111C1.71202 92.856 0 90.0859 0 87.0557V128H28H56V87.0557Z"
    />
    <path
      d="M12.4223 53.2111C9.71202 51.856 8 49.0859 8 46.0557V42L19.5777 47.7889C22.288 49.144 24 51.9141 24 54.9443V59L12.4223 53.2111Z"
    />
    <path
      d="M12.4223 71.2111C9.71202 69.856 8 67.0859 8 64.0557V60L19.5777 65.7889C22.288 67.144 24 69.9141 24 72.9443V77L12.4223 71.2111Z"
    />
    <path
      d="M12.4223 89.2111L24 95V90.9443C24 87.9141 22.288 85.144 19.5777 83.7889L8 78V82.0557C8 85.0859 9.71202 87.856 12.4223 89.2111Z"
    />
    <path
      d="M32 90.9443C32 87.9141 33.712 85.144 36.4223 83.7889L48 78V82.0557C48 85.0859 46.288 87.856 43.5777 89.2111L32 95V90.9443Z"
    />
    <path
      d="M32 72.9443V77L43.5777 71.2111C46.288 69.856 48 67.0859 48 64.0557V60L36.4223 65.7889C33.712 67.144 32 69.9141 32 72.9443Z"
    />
    <path
      d="M32 54.9443V59L43.5777 53.2111C46.288 51.856 48 49.0859 48 46.0557V42L36.4223 47.7889C33.712 49.144 32 51.9141 32 54.9443Z"
    />
  </svg>
</template>

<script src="./LogoCrustyBread.ts" lang="ts"></script>
<style src="./LogoCrustyBread.css" scoped></style>
