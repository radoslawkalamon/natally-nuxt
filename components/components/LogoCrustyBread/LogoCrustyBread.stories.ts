import { Meta, Story } from '@storybook/vue'
import LogoCrustyBread from './LogoCrustyBread.vue'

const meta: Meta = {
  title: 'Components / Logo Crusty Bread',
  component: LogoCrustyBread
}
export default meta

export const Default: Story = () => ({
  components: { LogoCrustyBread },
  template: '<LogoCrustyBread />'
})
