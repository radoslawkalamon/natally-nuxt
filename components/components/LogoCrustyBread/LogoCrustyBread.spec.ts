import LogoCrustyBread from '@/components/components/LogoCrustyBread/LogoCrustyBread.vue'
import { shallPassUnitSanityTest } from '@/devtools/jest.common.spec'

describe('Components / Logo Crusty Bread', () => {
  describe('Unit', () => {
    shallPassUnitSanityTest({ component: LogoCrustyBread })
  })
})
