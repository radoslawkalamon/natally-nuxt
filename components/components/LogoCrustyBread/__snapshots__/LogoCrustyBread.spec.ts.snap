// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components / Logo Crusty Bread Unit shall pass unit sanity test 1`] = `"<img alt=\\"Logo firmy Pieczywo Twarde\\" height=\\"128\\" src=\\"./images/crusty-bread.svg\\" title=\\"Logo firmy Pieczywo Twarde\\" width=\\"56\\">"`;
